app/views/my_facilities/drug_stocks/_form.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="modal fade" id=<%= "drug_report_modal_#{facility.id}" %> tabindex="-1" role="dialog" aria-labelledby="Drug Report" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content" style="text-align: left">
+      <div class="modal-header">
+        <h2 class="modal-title">Drug stock report</h2>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true"><i class="fas fa-times"></i></span>
+        </button>
+      </div>
+      <%= bootstrap_form_with(url: "/my_facilities/drug_stocks",
```

Comment:
Want to use rails' bidi paths here?

Reply:
This file is gone. The change has been made in `new.html.erb` now.